lib/utils/ensureOption.js

```diff
@@ -7,10 +7,18 @@
  */
 function defaults(defaultOptions) {
```

Comment:
What's that?

Reply:
There was a bug in this function that didn't accept falsy options when the defaultValue is true.

So for analytics, the default is true, but we want the ability to turn it off externally.

this line of code 
`let value = options[name] || defaultOptions[name] || defaultValue;	`

would always return true (because of the last defaultValue which is true)

Reply:
Sorry, still don't get it. 
Let's discuss on Sunday